src/Spd.Manager.Licence/ControllingMemberCrcAppManager.cs

```diff
@@ -280,20 +280,46 @@ private async Task<ChangeSpec> AddDynamicTasks(ControllingMemberCrcApplicationRe
         CancellationToken ct)
     {
         ChangeSpec changes = new();
+        //PeaceOfficerStatusChanged: check if Hold a Position with Peace Officer Status changed, create task with high priority, assign to Licensing CS team
+        PoliceOfficerRoleCode? originalRoleCode = originalApp.PoliceOfficerRoleCode == null ? null
```

Comment:
it should compare with contact, not originalApp

Reply:
replaced to compare with contact